packages/core/src/fhir-deduplication/deduplicate-fhir.ts

```diff
@@ -337,7 +337,9 @@ function removeDanglingReferences<T extends Resource>(entry: T, link: string): T
   if ("author" in entry) {
     if (entry.resourceType === "Composition") {
       entry.author = entry.author?.filter(author => author.reference !== link);
-      if (!entry.author.length) delete entry.author;
+      if (!entry.author.length) {
+        entry.author = [{ display: "No Known Author" }];
```

Comment:
author is a required field, but we don't need to necessarily provide a reference. Why not just do this as a display? It seems like a better option than: 
1. creating an empty practitioner placeholder
2. getting into complex logic that doesn't remove practitioners if its the only practitioner left in the composition

Reply:
Maybe we can do that as v0, but I think this is not a good long-term solution: it's just bypassing a requirement, it still doesn't prevent the composition from not having an author.

Reply:
right but the case is that the author of the composition is a practitioner resource that we drop because, for example, it has no name, address, location, telecom. So we could either 
1. not drop the practitioner, but then we would be leaving the kind of resource we said we were trying to get rid of 
2. do something like what we have here.